test/csuite/timestamp_abort/main.c

```diff
@@ -499,6 +495,9 @@ run_workload(uint32_t nth)
 
     thr = dcalloc(nth + 2, sizeof(*thr));
     td = dcalloc(nth + 2, sizeof(THREAD_DATA));
+    ts_data = dcalloc(nth, sizeof(uint64_t));
+    for (i = 0; i < nth; i++)
+        ts_data[i] = WT_TS_NONE;
```

Comment:
Do we need this? I believe `dcalloc` takes care of setting things to 0. I could be wrong but it's calling `calloc` internally.

Reply:
Yeah good point. We don't strictly need it, but I figured it was more correct to zero it to `WT_TS_NONE` rather than leave it as zero (even though they're the same thing, at least right now).

Reply:
"At least right now", who knows what `WT_TS_NONE` will be next! Let's keep it this way then :)